e2e/test/iothub/messaging/MessageReceiveE2EPoolAmqpTests.cs

```diff
@@ -102,6 +104,98 @@ public async Task Message_IoTHubSak_DeviceReceiveSingleMessage_MultipleConnectio
                 ConnectionStringAuthScope.IoTHub).ConfigureAwait(false);
         }
 
+        [LoggedTestMethod]
+        public async Task Message_DeviceSak_DeviceReceiveSingleMessageUsingCallback_MultipleConnections_Amqp()
+        {
+            await
```

Comment:
Why is await on its own line?

Reply:
this was the only way I could get the tabs to line up and look ok. Is it too weird?
Otherwise it ends up like this:
```csharp
await ReceiveMessageUsingCallbackPoolOverAmqpAsync(
	Client.TransportType.Amqp_Tcp_Only,
	PoolingOverAmqp.MultipleConnections_PoolSize,
	PoolingOverAmqp.MultipleConnections_DevicesCount)
	.ConfigureAwait(false);
```

I don't like how `.ConfigureAwait()` is not tabbed properly.

Reply:
I'd have done:

```csharp
    await ReceiveMessageUsingCallbackPoolOverAmqpAsync(
        Client.TransportType.Amqp_Tcp_Only,
        PoolingOverAmqp.MultipleConnections_PoolSize,
        PoolingOverAmqp.MultipleConnections_DevicesCount)
    .ConfigureAwait(false);
```

Reply:
I can update it in subsequent PRs, sure!